lib/redis_mutex/lock.ex

```diff
@@ -1,116 +1,84 @@
 defmodule RedisMutex.Lock do
   @moduledoc """
-  This module contains the actual Redis locking business logic. The `with_lock`
-  macro is generally the only function that should be used from this module, as it
-  will handle the logic for setting and removing key/values in Redis.
+  Defines the lock for RedisMutex.
   """
 
-  @default_timeout :timer.seconds(40)
-  @default_expiry :timer.seconds(20)
```

Comment:
is that defaulat documented anywhere?

Reply:
It's provided in the doc for `RedisMutex.with_lock/3`

https://github.com/podium/redis_mutex/blob/9ce361d4c638eb4fa696c7e5212e44ade73d4fb0/lib/redis_mutex.ex#L85-L97